exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -296,7 +295,7 @@ var questions = [
     }
   ),
   generateSchema(
-    'measures.questions.4.label',
+    'measures.questions.4.label.10am',
```

Comment:
Yes, I determine that here: https://github.com/CenterForOpenScience/exp-addons/pull/117/files#diff-9150b292474824ef67f55d3899fd974aR556

I specified a default so I could generate validators for that question. It's based on the index of the question in the list, so the exact label doesn't matter for validation.